docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,197 @@
+
+```csharp
+static void ShowBot(string message)
+{
+    string bot = $"\n        {message}";
+    bot += @"
+    __________________
+                      \
+                       \
+                          ....
+                          ....'
+                           ....
+                        ..........
+                    .............'..'..
+                 ................'..'.....
+               .......'..........'..'..'....
+              ........'..........'..'..'.....
+             .'....'..'..........'..'.......'.
+             .'..................'...   ......
+             .  ......'.........         .....
+             .    _            __        ......
+            ..    #            ##        ......
+           ....       .                 .......
+           ......  .......          ............
+            ................  ......................
+            ........................'................
+           ......................'..'......    .......
+        .........................'..'.....       .......
+     ........    ..'.............'..'....      ..........
+   ..'..'...      ...............'.......      ..........
+  ...'......     ...... ..........  ......         .......
+ ...........   .......              ........        ......
+.......        '...'.'.              '.'.'.'         ....
+.......       .....'..               ..'.....
+   ..       ..........               ..'........
+          ............               ..............
+         .............               '..............
+        ...........'..              .'.'............
+       ...............              .'.'.............
+      .............'..               ..'..'...........
+      ...............                 .'..............
+       .........                        ..............
+        .....
+";
+    Console.WriteLine(bot);
+}
+```
+
+### Test the tool
+
+You can run the project and see the output. Try these variations to run our application:
+
+```csharp
+dotnet run
+dotnet run -- "Hello from the bot"
+dotnet run -- hello from the bot
+```
+
+All arguments after the `--` delimiter are passed to our application.
+
+## Setup the global tool
+
+Before you can pack and distribute the application as a Global Tool, you need to modify the project file. Open the `botsay.csproj` file and add three new XML nodes to the `<Project><PropertyGroup>` node:
+
+- \<PackAsTool>  
+[REQUIRED] Indicates that the application will be packaged for install as a Global Tool.
+
+- \<ToolCommandName>  
+[OPTIONAL] An alternative name for the tool, otherwise the command name for the tool will be named after the project file. You can have multiple tools in a package, choosing a unique and friendly name will help differentiate from other tools in the same package.
```

Comment:
An alternative name for the tool, otherwise the command name for the tool will be named after the project file. ->
An alternative name for the tool. By default, the command name for the tool is the project file name.

remove future tense